packages/yarnpkg-fslib/sources/path.ts

```diff
@@ -58,6 +70,27 @@ npath.toPortablePath = toPortablePath;
 npath.contains = (from: NativePath, to: NativePath) => contains(npath, from, to);
 ppath.contains = (from: PortablePath, to: PortablePath) => contains(ppath, from, to);
 
+npath.FileURL = NativeFileURL;
+ppath.FileURL = PortableFileURL;
+
+npath.fromFileURL = (url: FileURL<NativePath>) => fileURLToPath(url);
+ppath.fromFileURL = (url: FileURL<PortablePath>) => fileURLToPosixPath(url);
+
+npath.toFileURL = (p: NativePath) => pathToFileURL(p);
+ppath.toFileURL = (p: PortablePath) => posixPathToFileURL(p);
+
+npath.PathBuffer = NativePathBuffer;
+ppath.PathBuffer = PortablePathBuffer;
+
+npath.fromPathBuffer = (buffer: PathBuffer<NativePath>) => buffer.toString(PATH_BUFFER_ENCODING);
+ppath.fromPathBuffer = (buffer: PathBuffer<PortablePath>) => buffer.toString(PATH_BUFFER_ENCODING);
+
+npath.toPathBuffer = (p: NativePath) => new NativePathBuffer(p);
+ppath.toPathBuffer = (p: PortablePath) => new PortablePathBuffer(p);
```

Comment:
I don't see where those functions are used 🤔

Reply:
They're for symmetry - this would look weird without them:

![image](https://user-images.githubusercontent.com/32596136/87985455-4c51b980-cae4-11ea-8b16-7086b78d7673.png)